E2ETests/Runner/AppDelegate.swift

```diff
@@ -0,0 +1,23 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
+import UIKit
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate {
+    var window: UIWindow?
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        let info = try! TestInfo() // crash if test info are missing or malformed
```

Comment:
🧠/ This may raise maintainability concerns. In case of this (and perhaps some other) failures, how would we know it from s8s run? The s8s test will go red (crash), but we will be missing any troubleshooting context. Waiting for CR being available in s8s is one way. I wonder if there are other. Thoughts?

Reply:
Yes, we can assume errors will be no often due to frequency of change in cofig (vs adding new scenarios). Let's see how it will work in wild 👍.